docs/release-process/CHECKLIST.md

```diff
@@ -0,0 +1,84 @@
+## Check list
+
+This section shows what things you need to do before creating a release. This is a manual process as CI does not test everything.
+
+The following example assumes we’re going from version 0.1.0 (`v0.1.0`) to
+0.2.0 (`v0.2.0`).
+
+- Checkout to old release tag
+  - e.g. `git checkout v0.1.0`
+
+- Build `lokoctl` binary from the last release
+  - e.g. `make build`
+  - Copy `lokoctl` binary to your assets directory.
+
+- Deploy `lokomotive` with old release
+  - e.g. `./lokoctl cluster apply`
+  - Deploy a single cluster(controller node) first.
+
+- Configure env variable `$KUBECONFIG` as per your assets directory location.
+
+- Make sure all nodes healthy and all Pod are running.
+  - e.g. `kubectl get po -A && kubectl get no`
+
+- Delete the old `lokoctl` binary.
+
+- Checkout to latest `master` branch. Make sure you have the latest code fetched and rebased.
+  - e.g. `git checkout master && git fetch origin && git rebase origin/master`
+  - Build lokoctl binary from latest master `make build`
+
+- Copy `lokoctl` binary to your assets directory and re-apply the cluster
+  - e.g. `./lokoctl cluster apply`
+
+- Make sure all nodes healthy and all Pod are running.
+  - e.g. `kubectl get po -A && kubectl get no`
+
+### Components test
+
+This sections checks if components work as desired. 
+
+- Check all certificates are valid
+  - e.g. `kubectl get certificates -A`
+  - Certificates for all your components are valid.
```

Comment:
What does valid mean in this case?

Reply:
Perhaps he meant `Ready`, as the column which shows up when you list the certificates.